### ISSUE TITLE
Massive spam to stderr on linux (gtk)

### ISSUE THREAD

**Issue by DarkFenX** [DarkFenX/Pyfa#282]
https://gist.github.com/DarkFenX/e5b5f16a6407425e403b
This is just from pyfa start. It spams even more during regular pyfa operation.

**Comment by blitzmann** [comment]
Been trying to get this to work on Debian, and it's been frustrating. There does not seem to be a debugging package with 3.0 (I owuld assume it would be called `python-wxgtk2.8-dbg`), which is required to import wx3 from `python-dbg` (which is, in turn, needed by `gdb` to *maybe* figure out where these errors are stemming from in the source).

Weird. Especially since sid has both `python-wxgtk2.8` and `python-wxgtk2.8-dbg`. And I don't know enough about it to attempt to compile my own debug version.

**Comment by blitzmann** [comment]
This is going to be a huge headache. I was able to compile the debug versions of python and wxpython to use in gdb, but I don't know how to set these GTK warnings to be fatal.

I will keep trying topost in the wxPython-users group. In the meantime, I have begun a painstaking process of following the logic and adding

    imprort sys
    sys.exit()

in different places to try to narrow down the issues. With this method I was able to find a source of major spam: when adding a page to chrome tabs, in the `ShowActive` function, it tries to set the size. Many times, the height is -4, which is too low. We can simply test if the value is less than -1 and if so simply replace it with -1.

I am not sure how widepread the problem is. there are 86 instances of `.SetSize()` throughout the gui. I don't know if it's just this instance that can have the problem (so far it seems like it) or if other ones might pop up. will need to look into this further.
